libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/feedback/FeedbackBottomSheet.java

```diff
@@ -123,7 +123,7 @@ private void initCountDownAnimation() {
     ObjectAnimator countdownAnimation = ObjectAnimator.ofInt(feedbackProgressBar,
       "progress", 0);
     countdownAnimation.setInterpolator(new LinearInterpolator());
-    countdownAnimation.setDuration(5000);
+    countdownAnimation.setDuration(10000);
```

Comment:
Maybe just pull this value out as a constant and later allow users to customize it?

Reply:
@cammace added the ability to adjust this when creating a new instance